php/conf.d/custom.ini

```diff
@@ -29,7 +29,7 @@ session.use_strict_mode = 1
 
 [opcache]
 opcache.enable_cli = On
-opcache.enable_file_override = On
+opcache.enable_file_override = Off
```

Comment:
This setting should not be deactivated in normal situations.

Perhaps, you should use instead a specific configuration file for the CLI SAPI?

Reply:
Will do the changes in a dedicated PR if the needs are still here